lib/rester/service/resource/params.rb

```diff
@@ -21,7 +21,7 @@ def initialize(opts={}, &block)
       # Whether or not validation will be done strictly (i.e., only specified
       # params will be allowed).
       def strict?
-        !!options[:strict]
+        !!options.fetch(:strict, true)
```

Comment:
what are your thoughts on initializing the `strict` option to true if it's not set in `initialize`?

`@options = opts.dup.tap { |o| o[:strict] = true unless o.has_key?(:strict) }.freeze `